tests/model_serving/model_runtime/vllm/fast/test_fast_runtime_inference.py

```
@@ -0,0 +1,144 @@
+    "deployment_mode": KServeDeploymentType.RAW_DEPLOYMENT,
+    "runtime_argument": FAST_SERVING_ARGUMENT,
+    "min-replicas": 1,
+}
+
+pytestmark = pytest.mark.usefixtures("skip_if_no_supported_accelerator_type", "valid_aws_config")
+
+
+def _assert_vllm_version_reported(isvc: InferenceService) -> None:
+    """Query the vLLM /version endpoint and verify it returns a non-empty version."""
+    url = get_exposed_isvc_url(isvc=isvc)
+    response = requests.get(f"{url}/version", verify=False, timeout=30)
```

Ruff (0.15.15)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)